docs/tracking-algorithms/tempest_extremes.rst

```diff
@@ -81,6 +81,11 @@ of one another:
         te.TEContour(var="zgdiff", delta=-6.0, dist=6.5, minmaxdist=1.0),
     ]
 
+    output_commands = [
+        te.TEOutputCommand(var="psl", operator="min", dist=0.0),
+        te.TEOutputCommand(var="orog", operator="max", dist=0.0),
+    ]
+
```

Comment:
Add something to lines 64-67 above to explain the the example now also adds some output fields (value of psl and orog at the candidate point).